Lib9c/Action/Factory/ClaimStakeRewardFactory.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using Libplanet;
+
+namespace Nekoyume.Action.Factory
+{
+    public static class ClaimStakeRewardFactory
+    {
+        // NOTE: This method does not return a type of `ClaimStakeReward1`.
+        //       Because it is not obsoleted yet.
+        public static IClaimStakeReward CreateByBlockIndex(
```

Comment:
I think the method name `ClaimStakeReward` is the best one for both functions because it's easy to use with the same name and can shorten code: `ClaimStakeRewardFactory.CreateByBlockIndex` > `ClaimStakeReward`

Reply:
@U-lis I also prefer shorter code. However, you don't want to miss out on necessary information.
Can you explain the advantages of making method names shorter than lengthy code but explicitly using them?